pkg/utils/utils_test.go

```diff
@@ -46,7 +46,7 @@ func TestObjectHash(t *testing.T) {
 			input: v1.Secret{Data: map[string][]byte{
 				"xx": []byte("yyy"),
 			}},
-			want: "a9fe13fd43b20829b45f0a93372413dd",
```

Comment:
interesting, so a version bump results in different object hash :thinking:

Reply:
so all external secrets are then reconciled after the bump

Reply:
Yes... not sure why though, but this is happening. AFAIK all external secrets are reconciled anyways after an upgrade. Are you thinking some additional issues will happen?

Reply:
Yes, they're reconciled, but the hash would prevent the controller from actually syncing it.
I don't think this should be an issue :thinking:

Reply:
I'll run some manual upgrade tests before finishing this bump...

Reply:
Not an issue :)